Makefile

```diff
@@ -13,6 +6,16 @@ pip := $(virtualenv_bin)/pip
 run_dir := $(realpath $(makefile_dir)/..)/.run
 conf_dir := $(realpath $(makefile_dir)/..)/conf
 
+export PATH := $(wrapper_bin):$(virtualenv_bin):$(PATH)
```

Comment:
Why is this needed? Can you add a code comment?

Reply:
This is needed as depending on your way of managing python and your `PATH` the PATH you create is rewritten by the shell user's configuration and not using your virtualenv or your python.